app/models/role.rb

```diff
@@ -38,6 +38,14 @@ def current_holder
       .first
   end
 
+  def biography
```

Comment:
```suggestion
  def current_holder_biography
```

It might be better to call it this just because a role doesn't have a biography on its own.

Reply:
Ok sounds good to me 👍